### DISCUSSION TITLE
More details about v1.14.5 release date

### DISCUSSION THREAD

**Discussion by Alirezahaghgoo1** [dogecoin/dogecoin#2650] [General]
Hello Dogecoin devs,
As we are all excited about the new update and transaction fee reduction, we are following the process of updating on Github step by step. but can we kindly ask for an estimated release date for v1.14.5, please?
We appreciate the hard work of the team!
Thank you devs!

**Comment by patricklodder** [comment]
I've opened an "open items" issue at #2655 for better transparency. It's all reviews, documentation and simple maintenance tasks now, so we're talking a couple of days. I'd say this weekend or early next week, bar any new bugs.
